js/form.js

```diff
@@ -1,5 +1,80 @@
+import '../pristine/pristine.min.js';
 import { disableForm, enableForm } from './utils.js';
 const form = document.querySelector('.ad-form');
 
-disableForm(form);
-enableForm(form);
+const disableAdForm = () => {
+  disableForm(form, 'ad-form--disabled');
+};
+
+const enableAdForm = () => {
+  enableForm(form, 'ad-form--disabled');
+};
+
+export { disableAdForm, enableAdForm };
+
+form.setAttribute('action', 'https://26.javascript.pages.academy/keksobooking');
+form.setAttribute('method', 'POST');
+
+const pristine = new Pristine(form, {
+  classTo: 'ad-form__element',
+  errorTextParent: 'ad-form__element',
+  errorTextClass: 'ad-form__error-text',
+});
+
+const titleField = form.querySelector('#title');
+titleField.setAttribute('requred', '');
+
+const maxSymbols = {
```

Comment:
В целом, я не особо стараюсь заморачиваться над критериями до того момента, как проект будет готов, но если ты предполагаешь тут перечисление, то с большой буквы, ключи в верхнем регистре + почему название содержит max? Максимальное минимальное символы `maxSymbols.min` и максимальное максимальное символы `maxSymbols.max`
```suggestion
const TitleLengthRange = {
   MIN: 30,
   MAX: 100
}
```
Лучше посмотри в сторону чего-то более понятного, например перечисление (псевдо enum) длина строки, с двумя значениями минимальным и максимальным, при использовании все более чем очевидно: title.length >= TitleLengthRange.MIN && title.length <= TitleLengthRange.MAX или типо того, не обязательно так (Можно TitleValidLengthили ещё что-то и т.д.), просто код должен максимально читаться